rpcserver.go

```diff
@@ -1392,6 +1400,13 @@ func (s *rpcServer) validateOrder(o order.Order, acct *account.Account,
 			"the same time")
 	}
 
+	if o.Details().AuctionType == order.BTCOutboundLiquidity &&
+		o.Details().Units != 1 {
+
+		return fmt.Errorf("to participate in the outbound liquidity " +
+			"market the order amt should be exactly 100k sats")
```

Comment:
This error message talks about amount whereas the if conditional examines units. I expected something like:

```go
if o.Details().AuctionType == order.BTCOutboundLiquidity &&
	o.Details().Amt != order.BaseSupplyUnit {
        return fmt.Errorf("to participate in the outbound liquidity " +
			"market the order amt should be exactly the base supply amount (%v sats)", order.BaseSupplyUnit)
}
```